tests/unit/acl.tcl

```diff
@@ -984,16 +984,45 @@ start_server [list overrides [list "dir" $server_path "acl-pubsub-default" "allc
         set e
     } {*NOPERM*set*}
 
+    test {SLAVAK 1} {
```

Comment:
great test name!
but still let's change it...

Reply:
Whoopsie :smile: